godtools/App/Features/OptInNotification/Presentation/OptInNotification/OptInNotificationViewModel.swift

```diff
@@ -14,6 +14,8 @@ import UserNotifications
 
 class OptInNotificationViewModel: ObservableObject {
 
+    var isInitialPrompt: Bool = false
```

Comment:
Hey @JasonBennett41 looks like this will need to be a Published property since the SwiftUI View is accessing this property.